ibis/pyspark/compiler.py

```diff
@@ -343,110 +346,77 @@ def fn(col):
 
 @compiles(ops.Any)
 def compile_any(t, expr, scope, context=None, **kwargs):
-    def fn(col):
-        if 'window' in kwargs:
-            window = kwargs['window']
-            return F.max(col).over(window)
-        else:
-            return F.max(col)
-
-    return compile_aggregator(t, expr, scope, fn, context, **kwargs)
+    return compile_aggregator(t, expr, scope, F.max, context, **kwargs)
 
 
 @compiles(ops.NotAny)
-def compile_notany(t, expr, scope, context=None, **kwargs):
-    def fn(col):
-        if 'window' in kwargs:
-            window = kwargs['window']
-            return ~F.max(col).over(window)
-        else:
+def compile_notany(t, expr, scope, *, context=None, window=None, **kwargs):
+    # The code here is a little ugly because the translation are different
```

Comment:
@icexelloss could you provide a docstring here using numpy format? https://numpydoc.readthedocs.io/en/latest/format.html